internal_workflow_testsuite.go

```diff
@@ -604,6 +618,10 @@ func (env *testWorkflowEnvironmentImpl) CompleteActivity(taskToken []byte, resul
 	return nil
 }
 
+func (env *testWorkflowEnvironmentImpl) getHostEnv() *hostEnvImpl {
+	return env.testSuite.hostEnv
```

Comment:
Let's keep this discussion on your PR. I added some thoughts on global vs keeping the private env there. This shouldn't be a blocker on this one. We can make the change in parallel.